.github/workflows/pr-tests-e2e-contracts.yml

```diff
@@ -58,7 +58,7 @@ jobs:
   tests-e2e-contracts-beta:
     name: Test (Chrome Beta)
     runs-on: buildjet-8vcpu-ubuntu-2204
-    timeout-minutes: 10
+    timeout-minutes: 20
     continue-on-error: true
     steps:
```

Comment:
- Test times have decreased with the latest fixes, so we should try keeping this number at max 15 if possible

![CleanShot 2024-12-10 at 15 15 56](https://github.com/user-attachments/assets/2fe24549-6947-49cf-83f8-a350a2fbf2a2)